squema.sql

```diff
@@ -2,11 +2,13 @@
 
 CREATE TABLE animals
 (
-    id integer NOT NULL,
+    id integer NOT NULL IDENTITY(1,1),
     name text NOT NULL,
     date_of_birth date NOT NULL,
     escape_attemps integer,
```

Comment:
- [ ] Kindly correct this typo use `escape_attempts` instead of `escape_attemps` because I am getting errors when trying to insert new data in your table, this is the error I am getting :point_down: 

![image](https://user-images.githubusercontent.com/65068771/155236012-a5a8a49b-4302-417b-bae9-adabd90b22e3.png)